### PR TITLE
Update Getting Started pages

### PR DESCRIPTION
**Pull request by amolenaar**
### PR Type
What kind of change does this PR introduce?

- [ ] Bug fix
- [ ] Feature
- [ ] Chore (refactoring, formatting, local variables, other cleanup)
- [x] Documentation content changes

### What is the current behavior?

Getting started documentation is on https://gaphor.org. It's slowly becoming outdated.

### What is the new behavior?

Move Getting Started docs to https://gaphor.readthedocs.io. We can more easily keep it up to date. It's also a smoother integration with the Feature sections.

### Other information

* Renamed the default diagram in the _Generic_ template to `New diagram`.

### CODE REVIEW

**Review by danyeaw — APPROVED** [review]
I love this! I added some additional text and fixed up some spelling and grammar. I think adding a few more pictures would also help with the explanations.

File: docs/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+namely:
+
+- Navigator
+- Diagrams
+- Diagram Element Toolbox
+- Properties pane
+
+Each section has its own specific function.
+
+## Navigator
```
[the remark this answers is not in the document]
Comment (amolenaar) [reply docs/getting_started.md:41]: I like Model Browser. :+1:

File: docs/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+- **RAAML:** *Risk Analysis and Assessment Modeling language* template
+- **C4 Model:** *a lean graphical notation technique for modelling the architecture of software systems* template
+
+Once the model interface is loaded you'll see the modeling interface.
+
+![new model](images/getting-started-new-model.png)
+
+The layout of the Gaphor interface is divided into four sections,
+namely:
+
```
[the remark this answers is not in the document]
Comment (amolenaar) [reply docs/getting_started.md:33]: I added numbers to the picture.

**Comment by amolenaar** [comment]
@danyeaw i renamed Navigator to Model Browser and added numbers to the getting-started image.

**Pull request merged**